sdc_visualization/server.py

```diff
@@ -284,11 +290,24 @@ def get_cdi_ids():
     logger.info('variable  cdi-id {}'.format(var_name))
     cdi_ids = netCDF4.chartostring(ds.variables[var_name][:])
     return cdi_ids
+"""
+@functools.lru_cache()
+def get_cdi_id_var(ds):
+    
+    for var_name, var in ds.variables.items():
+        if var.long_name == "LOCAL_CDI_ID":
```

Comment:
See commented function above. Can't always be sure that LOCAL_CDI_ID is the var name we need.
Discuss if OK for now, and add TODO to remind for later?